mlflow/llama_index/serialize_objects.py

```diff
@@ -116,14 +119,27 @@ def dict_to_object(object_representation: Dict[str, Any]) -> object:
         object_class = getattr(module, class_name)
 
         for k, v in OBJECT_DICT_METHOD_MAP.items():
-            if isinstance(object_class, k):
```

Comment:
`isinstance` only works between isntance and class and we must use `issubsclass` here.